lua/entities/gmod_wire_expression2/core/player.lua

```diff
@@ -381,7 +381,7 @@ local steamfriends = {}
 concommand.Add("wire_expression2_friend_status", function(ply, command, args)
 	local friends = {}
 
-	for index in args[1]:gmatch("[^,]") do
+	for index in args[1]:gmatch("[^,]+") do
 		local n = tonumber(index)
 		if not n then return end
 		table.insert(friends, Entity(index))
```

Comment:
Oh wow I didn't even notice that, seems that it converted itself when being used in though as it still worked. I'll change that.